graviton/blackbox.py

```diff
@@ -1,6 +1,7 @@
 from base64 import b64decode
```

Comment:
TODO: consider breaking up into a small PR that first just move classes around and breaks this humongous glass into more palatable pieces